# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libreoffice/hotfix-no-rm-rf-sysroot.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Thanks, but no thanks - we prefer all our sysroot not to be deleted, ...

--- libreoffice-7.6.6.3/solenv/bin/ooinstall.vanilla	2024-05-22 09:34:34.769909488 +0200
+++ libreoffice-7.6.6.3/solenv/bin/ooinstall	2024-05-22 09:36:16.214900493 +0200
@@ -82,8 +82,8 @@
 # it creates symlinks from the original path to /usr/share; they are not valid with $DESTDIR
 # and potential second ooinstall call fails with strange error messages if the old tree is still there
 if ($destdir && "$ENV{DESTDIR}" ne "/" && -d "$ENV{DESTDIR}") {
-    print "Cleaning destdir...\n";
-    system ("rm -rf \"$ENV{DESTDIR}\"") && die "Failed to clean up destdir: $!";
+    #print "Cleaning destdir...\n";
+    #system ("rm -rf \"$ENV{DESTDIR}\"") && die "Failed to clean up destdir: $!";
 }
 
 print "Running LibreOffice installer\n";
